mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -424,15 +424,19 @@ function triggerVolume() {
     volumeChanging = 0;
 }
 
-function toggleMute() {
-    mopidy.mixer.getMute().then(function(mute) {
-        mopidy.mixer.setMute(!mute);
+function setMute(nwmute) {
```

Comment:
Yep, oops, you are of course totally right. We should look at making it more obvious at some point in the future.